src/DetailsView/reports/report-generator-impl.ts

```diff
@@ -10,7 +10,7 @@ import { ReportGenerator } from './report-generator';
 import { ReportHtmlGenerator } from './report-html-generator';
 import { ReportNameGenerator } from './report-name-generator';
 
-export class ReportGeneratorV2 implements ReportGenerator {
+export class ReportGeneratorImpl implements ReportGenerator {
```

Comment:
Is separating the Impl and the header interface providing any value now that there's only one impl?

Reply:
Updated. Directly impacts the `DetailsViewInitializer`, `IssuesTable` and `DetailsViewCommandBar`.